Test/NativeUnitTest/NativeUnitTest.vcxproj

```diff
@@ -93,15 +93,21 @@
       <WarningLevel>Level3</WarningLevel>
       <Optimization>Disabled</Optimization>
       <SDLCheck>true</SDLCheck>
-      <AdditionalIncludeDirectories>$(VCInstallDir)UnitTest\include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>$(VCInstallDir)UnitTest\include;C:\Program Files (x86)\Typemock\Isolator++\include\ForTests;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```

Comment:
Ah, what if on my PC this installed in folder fuck_the_project_file instead of C:\Program Files (x86)?

Reply:
Uhh, I guess I will address this in a later PR